general/overlay/etc/wireless

```diff
@@ -83,4 +83,42 @@ if [ "$1" = "l716-t31-camhi" ]; then
 fi
 
 
+# T31 ATBM603X on SDIO bus mmc0
+if [ "$1" = "atbm603x-t31-mmc0" ]; then
+	# PB10 in wifi mmc1 is in the pull-down state by default, it needs to be set to a high-impedance state
+	devmem 0x10011128 32 0x400
+	# set wifi mmc1 clk drive capability to 8mA
+	devmem 0x10011134 32 0x20000
+
+	# "insert" the bus mmc0 device
+	if [ "$(cat /sys/devices/platform/jzmmc_v1.2.0/present)" = "N" ]; then
```

Comment:
this is important, it's not an "identification" procedure, just a mechanism to make the mmc (SDIO) driver enumerate the devices if it haven't.

same on line 112